fecfile_validate_js/scripts/buildSchemaModules.js

```diff
@@ -3,44 +3,56 @@
  * This script is run as part of the build process and at postinstall.
  */
 
-import * as fs from 'fs';
-import glob from 'glob';
-import * as path from 'path';
-import * as url from 'url';
+import * as fs from "fs";
+import glob from "glob";
+import * as path from "path";
+import * as url from "url";
 
-import Ajv from 'ajv';
-import standaloneCode from 'ajv/dist/standalone/index.js';
+import Ajv from "ajv";
+import standaloneCode from "ajv/dist/standalone/index.js";
+
+const versions = ["8.4", "8.5"];
 
 const dir = path.dirname(url.fileURLToPath(import.meta.url));
 fs.mkdirSync(path.join(dir, `../dist`), { recursive: true });
-const files = glob.sync(path.join(dir, '../../schema/*.json'));
 
-for (const file of files) {
-  const schema = JSON.parse(fs.readFileSync(file, 'utf-8'));
+for (const version of versions) {
```

Comment:
@toddlees @sasha-dresden I wonder if we want to generate all schemas for every version, or perhaps just the schemas we're expecting to be different? (ie, are we going to switch to 8.4 wholesale or piecemeal?)